future.tex

```diff
@@ -108,6 +33,38 @@ \subsection{Summary}
 	}
 \end{table}
 
+The next table lists different nations in the EU and their nuclear program plans.
+The attitude is divided into five, from very negative to very positive, where 
+very negative means a complete shutdown of all nuclear facilities and 
+very positive means a rigorous expansion of nuclear power in the country.
+
+\begin{table}[h]
+	\centering
+	\caption {Future Nuclear Programs of EU Nations \cite{world_nuclear_association_nuclear_2017}}
```

Comment:
The caption often comes at the end of tables, but it's up to you, regarding style.